project_name/settings/staging.py

```diff
@@ -1,11 +1,11 @@
-from pennlabs.settings.production import *
+from {{ project_name }}.settings.base import *
 
 
 DEBUG = True
 
 PLATFORM_ACCOUNTS.update(
     {
-        'REDIRECT_URI': os.environ.get('LABS_REDIRECT_URI', 'https://{}/accounts/callback/'.format(BACKEND_DOMAIN)),
+        'REDIRECT_URI': os.environ.get('LABS_REDIRECT_URI', f'https://{BACKEND_DOMAIN}/accounts/callback/'),
```

Comment:
Not a huge fan of  f-strings since they can execute arbitrary python code.

Reply:
oh for real? yike

Reply:
f-strings can't change arbitrary python code if the thing you pass into them is a string. They can only execute arbitrary python code if you said something like `f'https://{print(hi)}`. Since `BACKEND_DOMAIN` is a string, we don't need to be worried about this. We should be using f-strings here.

Reply:
Agreed. After testing, it doesn't appear there's a way to execute code through an f-string.